### PR TITLE
Hcap 708 bugfixes

### PR DESCRIPTION
**Pull request by dbayly-freshworks**
- Hired or withdrawn participants can no-longer reconfirm their interest 770
- Added a success page after confirm interest to match the designs 775
- Set the home button to return logged in users to the participant landing 

Screen Shot 2021-07-20 at 4 57 07 PM

### CODE REVIEW

File: client/src/pages/private/ParticipantEOI.js [hunk 1]
```diff
@@ -283,18 +278,17 @@ export default () => {
...
-                          setOpenConfirmInterest(true);
-                        }}
-                      >
-                        Reconfirm interest
-                      </Button>
-                    </Grid>
-                  )}
+                  <Grid item>
+                    <Button
+                      disabled={disableWithdraw}
```
Comment (fwpushan) [inline client/src/pages/private/ParticipantEOI.js:283]: What happen when user already reconfirmed?

**Review by fwpushan — COMMENTED** [review]
LGTM. Just ask for one clarification. Please check.

File: client/src/pages/private/ParticipantActionSuccess.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      return history.push(Routes.ParticipantLanding);
+    }
+  };
+
+  return (
+    <Page hideEmployers={!window.location.hostname.includes('freshworks.club')}>
+      <Grid container spacing={4} justify='center' alignContent='center' direction='column'>
+        <Grid className={classes.textBoxes} item>
+          <Typography variant={'h2'}>Successfully Confirmed Interest</Typography>
+          <Typography>You have successfully confirm your interest and status with HCAP</Typography>
```
Comment (Zenen-freshworks) [inline client/src/pages/private/ParticipantActionSuccess.js:41]:
```suggestion
          <Typography>You have successfully confirmed your interest and status with HCAP</Typography>
```

File: client/src/pages/private/ParticipantActionSuccess.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        </Grid>
+        <Grid className={classes.buttonBoxes} item>
+          <Button className={classes.button} variant={'contained'} onClick={handleBackCLick}>
+            View Expression of of Interest
```
Comment (Zenen-freshworks) [inline client/src/pages/private/ParticipantActionSuccess.js:45]:
```suggestion
            View Expression of Interest
```

File: client/src/pages/private/ParticipantActionSuccess.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  textBoxes: {
+    marginTop: '200px',
+  },
+}));
+
+export default () => {
+  const classes = useStyles();
+  const { id } = useParams();
+  const history = useHistory();
+  const handleBackCLick = () => {
```
Comment (Zenen-freshworks) [inline client/src/pages/private/ParticipantActionSuccess.js:28]:
```suggestion
  const handleBackClick = () => {
```

File: client/src/pages/private/ParticipantActionSuccess.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+  return (
+    <Page hideEmployers={!window.location.hostname.includes('freshworks.club')}>
+      <Grid container spacing={4} justify='center' alignContent='center' direction='column'>
+        <Grid className={classes.textBoxes} item>
+          <Typography variant={'h2'}>Successfully Confirmed Interest</Typography>
+          <Typography>You have successfully confirm your interest and status with HCAP</Typography>
+        </Grid>
+        <Grid className={classes.buttonBoxes} item>
+          <Button className={classes.button} variant={'contained'} onClick={handleBackCLick}>
```
Comment (Zenen-freshworks) [inline client/src/pages/private/ParticipantActionSuccess.js:44]:
```suggestion
          <Button className={classes.button} variant={'contained'} onClick={handleBackClick}>
```

**Review by Zenen-freshworks — DISMISSED** [review]
Looks good! I see a few copy nitpicks, but the logic looks sound

File: client/src/pages/private/ParticipantEOI.js (hunk 1 above)
Comment (dbayly-freshworks) [reply client/src/pages/private/ParticipantEOI.js:283]: We should leave this be. From talking with Kendall, the client wants users to have the ability to reconfirm interest as much as they want.

**Review by Zenen-freshworks — APPROVED** [review]

**Review by fwpushan — APPROVED** [review]

**Pull request merged**